index.css

```diff
@@ -64,8 +64,8 @@ a:link {
     margin-left: -5px;
     font-size: 1rem;
     border: none;
-    width: 20rem;
-    padding-left: 5rem;
+    width: 24rem;
+    
     background-color: var(--grey-color);
     border-top-right-radius: 0.5rem;
     border-bottom-right-radius: 0.5rem;
```

Comment:
changes: use px units for less than 1rem.

Reply:
I tried but then it exceeds 10px, and using px for >10 is also not good.